commons/src/main/java/org/frankframework/util/ClassUtils.java

```diff
@@ -256,10 +259,22 @@ public static <T> T convertToType(Class<T> type, String value) throws IllegalArg
 	}
 
 	private static Object convertToTypeRawTyped(Class<?> type, String value) throws IllegalArgumentException {
+		if (value == null) {
+			return null;
+		}
 		//Try to parse the value as an Enum
 		if(type.isEnum()) {
 			return parseAsEnum(type, value);
 		}
+		// Unbox an array to its component type. Convert string input to values. Put back into an array with the right type
+		if (type.isArray()) {
+			List<Object> list = Arrays.stream(value.split(","))
```

Comment:
Hier kan je beter `StringUtil.splitToStream` gebruiken.

Reply:
Was ik me niet bewust van. Maar is idd handig. Thanks